test/integration/manual_test.rb

```diff
@@ -0,0 +1,65 @@
+require "test_helper"
+
+class ManualTest < ActionDispatch::IntegrationTest
+  test "page renders correctly" do
+    setup_and_visit_content_item("content-design")
+
+    assert_has_component_title(@content_item["title"])
+    assert page.has_text?(@content_item["description"])
+  end
+
+  test "renders metadata" do
+    setup_and_visit_content_item("content-design")
+
+    assert_has_metadata(
```

Comment:
I often wonder if we should somehow reference or link in a comment to the [govuk-content-schema](https://github.com/alphagov/govuk-content-schemas/blob/b3c10be62e86ef6ee4fb482f75dbd4c4863ebae7/examples/manual/frontend/content-design.json) this data comes from. It can sometimes be a little opaque to trace from this test => gds-api-adapters => govuk-content-schemas. More a thought than a suggestion!

Reply:
Yeah, it's a general faff rather than anything for this PR in particular to solve!